game_event_player/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from game_event.models import GameEvent
+from player.models import Player
+
+
+class GameEventPlayer(models.Model):
+    game_event_id = models.ForeignKey(GameEvent, on_delete=models.CASCADE)
+    user_id = models.ForeignKey(Player, on_delete=models.CASCADE)
```

Comment:
Hi sorry to by a bit late to the party.

question why we don't use https://docs.djangoproject.com/en/4.2/ref/models/fields/#manytomanyfield in the game_event model. 

django will create the connection table for you